docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+Now you will pull them together with the [`objectfilter`](https://app.viam.com/module/felixreichenbach/object-filter) {{< glossary_tooltip term_id="module" text="module" >}}.
+This module takes a vision service (in this case, your hard hat detector) and applies it to your webcam feed.
+It outputs a stream with bounding boxes around the hard hats (and people without hard hats) in your camera's view so that you can see the detector working.
+This module also filters the output so that later, when you configure data management, you can save only the images that contain people without hard hats.
+
+1. Navigate to the **Components** subtab of your machine's **Config** tab.
+
+2. Click **Create component**.
+   Start typing `objectfilter` and select **camera / objectfilter** from the results.
+
+3. Name your filtering camera something like `objectfilter-cam` and click **Create**.
+
+4. Paste the following into the attributes box:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "filter_data": true,
+     "camera": "my_webcam",
+     "vision_services": ["yolo"],
+     "labels": ["Hardhat", "NO-Hardhat"],
+     "confidence": 0.5,
+     "display_boxes": true
+   }
+   ```
+
+   If you named your detector something other than "yolo," edit the `detector_name` line accordingly.
+   You can also edit the confidence threshold.
+   If you change it to `0.2` for example, you will see detections of over 20% confidence.
+
+   Setting the `filter_data` attribute to `true` means that later, when you configure data capture on this camera, only images that contain one or more of the labels will be captured.
+
+   Your `objectfilter` camera configuration should now resemble the following:
+
+   {{<imgproc src="/tutorials/helmet/filtercam-config.png" resize="x1100" declaredimensions=true alt="The detector_cam config panel in the Viam app." >}}
+
+5. Click **Save config** at the bottom of the screen.
+
+### Test the detector
+
+Now that the detector is configured, it's time to test it!
+
+1. Navigate to the **Control** tab.
+
+2. Click the **objectfilter_cam** panel to open your detector camera controls.
+
+3. Toggle **View objectfilter_cam** to the "on" position.
+   This displays a live feed from your webcam with detection bounding boxes overlaid on it.
+
+4. The detector can recognize humans with and without hard hats on.
+   Try positioning yourself in front of the camera without anything on your head, and then with a helmet on, and watch the bounding boxes appear.
+
+   {{<imgproc src="/tutorials/helmet/no-hard-hat1.png" resize="x1100" declaredimensions=true alt="A person with no hard hat on, with a bounding box labeled No-Hardhat around her head." >}}
+
+## Configure data capture and sync
+
+Viam's built-in [data management service](/data/) allows you to, among other things, capture images and sync them to the cloud.
+Configure data capture on the `objectfilter` camera to capture images of people without hard hats:
```

Comment:
Or somethign like that, we want to explain why people would want to capture this
```suggestion
Configure data capture on the `objectfilter` camera to capture images of people without hard hats, so that you can see who was not wearing a hard hat:
```

Reply:
```suggestion
For this project, you will capture images of people without hard hats so that you can see who wasn't wearing one, and so that you can trigger notifications when these images are captured and synced.
Configure data capture on the `objectfilter` camera to capture images of people without hard hats:
```

Reply:
I like it!1